docs/source/run_model_in_simulation.rst

```diff
@@ -0,0 +1,204 @@
+Run a model in gem5-Aladdin simulation
+======================================
+
+Following the tutorial in :ref:`label_build_python_model`, now we are able to
+create a DL model using the SMAUG Python APIs. In this tutorial, we will
+proceed to introduce the steps of running a model in gem5-Aladdin simulation.
```

Comment:
At this time, it's worth reiterating that the user should be using the Docker image.

Reply:
Done.